cluster/crds/pkg.crossplane.io_controllerconfigs.yaml

```diff
@@ -1219,6 +1230,17 @@ spec:
                         description: GMSACredentialSpecName is the name of the GMSA
                           credential spec to use.
                         type: string
+                      hostProcess:
```

Comment:
This is a new field so we'll either need to support translating it or craft our own struct with it omitted.

Reply:
It's part of pod security context and that field seems to be set as is [here](https://github.com/crossplane/crossplane/blob/8c48d4c/internal/controller/pkg/revision/deployment.go#L151). So, I think it's handled?

Reply:
Ah good call -- thanks!